homeassistant/components/device_tracker/bmw_connected_drive.py

```diff
@@ -42,7 +42,8 @@ def update(self) -> None:
         attrs = {
             'trackr_id': dev_id,
             'id': dev_id,
-            'name': self.vehicle.name
+            'name': self.vehicle.name,
```

Comment:
Remove this, it's duplicate info from `host_name`.

Reply:
fixed, see also #12999